.github/workflows/main-cmake.yml

```diff
@@ -253,20 +280,20 @@ jobs:
               name: zip-vs2022_cl-${{ inputs.build_mode }}-binary
               path: ${{ runner.workspace }}/build/HDF5-*-win64.zip
               if-no-files-found: error # 'warn' or 'ignore' are also available, defaults to `warn`
-        if:  ${{ (matrix.os == 'windows-latest') && (inputs.thread_safety != 'TS') && ( inputs.build_mode != 'Debug') }}
+        if:  ${{ (matrix.os == 'windows-latest') && (inputs.thread_safety != 'TS') && (inputs.concurrent != 'CC') && ( inputs.build_mode != 'Debug') }}
```

Comment:
Should there be binary versions for testing concurrent installs? That is what these steps are for - see the cmake-bintest.yml file.

Reply:
No, I'm excluding the binaries from concurrent installs, same as for threadsafe builds.

Happy to enable them, if you'd prefed.